lib/games.rb

```diff
@@ -101,6 +101,19 @@ def average_goals(stat)
     goals_collection
   end
 
+  def average_goals(stat)
+    goals_collection = {}
+    stat_collect(stat).each do |collection|
+      collection = collection.to_sym
+      goals_collection[collection] = 0
```

Comment:
Do we actually need this line?

Reply:
I don't think we do, we can try it without and see if it works.